src/main/java/uk/gov/justice/digital/delius/jwt/JwtValidator.java

```diff
@@ -32,8 +31,6 @@ public void validateJwt(JoinPoint joinPoint) {
 
         ArrayList<Object> argsList = Lists.newArrayList(joinPoint.getArgs());
 
-        log.info("Received call to {} with arguments {}", joinPoint.getSignature().getName(), argsList.stream().map(Object::toString).collect(Collectors.joining(", ")));
```

Comment:
Bearer token for the non secure endpoints were being logged

Reply:
:+1: